bundle/manifests/sandboxed-containers-operator.clusterserviceversion.yaml

```diff
@@ -13,26 +13,26 @@ metadata:
         }
       ]
     capabilities: Seamless Upgrades
-    createdAt: "2024-10-04T09:21:08Z"
+    createdAt: "2024-11-18T19:07:31Z"
     features.operators.openshift.io/disconnected: "true"
     features.operators.openshift.io/fips-compliant: "false"
     features.operators.openshift.io/proxy-aware: "false"
     features.operators.openshift.io/tls-profiles: "false"
     features.operators.openshift.io/token-auth-aws: "false"
     features.operators.openshift.io/token-auth-azure: "false"
     features.operators.openshift.io/token-auth-gcp: "false"
-    olm.skipRange: '>=1.1.0 <1.8.0'  ## OSC_VERSION
+    olm.skipRange: '>=1.1.0 <1.8.0'
```

Comment:
Just a note that @c3d's initiative to track locations where we bump the OSC version doesn't fly with `make bundle`... unfortunate side effect of the Operator Framework policy to track generated files :disappointed: